pkg/monitoring/observability/rules/alerts/cluster_alerts.go

```diff
@@ -36,7 +48,7 @@ func clusterAlerts() []promv1.Rule {
 		},
 		{
 			Alert: "NodeNetworkInterfaceDown",
-			Expr:  intstr.FromString("count by (instance) (node_network_up{device!~\"veth.+|tunbr\"} == 0) > 0"),
+			Expr:  intstr.FromString(fmt.Sprintf("count by (instance) (node_network_up{device!~\"%s\"} == 0) > 0", strings.Join(ignoredInterfacesForNetworkDown, "|"))),
```

Comment:
lrt's get rid of the ugly `\"`. either by
```suggestion
			Expr:  intstr.FromString(fmt.Sprintf(`count by (instance) (node_network_up{device!~"%s"} == 0) > 0`, strings.Join(ignoredInterfacesForNetworkDown, "|"))),
```

or by

```suggestion
			Expr:  intstr.FromString(fmt.Sprintf("count by (instance) (node_network_up{device!~%q} == 0) > 0", strings.Join(ignoredInterfacesForNetworkDown, "|"))),
```

Reply:
updated